--- reno-2.11.3/reno/tests/test_cache.py.orig	2019-04-04 05:59:54.000000000 +0200
+++ reno-2.11.3/reno/tests/test_cache.py	2020-01-13 20:36:08.997885009 +0100
@@ -15,7 +15,10 @@
 import fixtures
 import textwrap
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from reno import cache
 from reno import config
--- reno-2.11.3/reno/tests/test_config.py.orig	2019-04-04 05:59:54.000000000 +0200
+++ reno-2.11.3/reno/tests/test_config.py	2020-01-13 20:36:28.684445025 +0100
@@ -21,7 +21,10 @@
 from reno import main
 from reno.tests import base
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 
 class TestConfig(base.TestCase):
--- reno-2.11.3/reno/tests/test_create.py.orig	2019-04-04 05:59:54.000000000 +0200
+++ reno-2.11.3/reno/tests/test_create.py	2020-01-13 20:36:44.131028010 +0100
@@ -14,7 +14,10 @@
 
 import fixtures
 import io
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from reno import create
 from reno.tests import base
--- reno-2.11.3/reno/tests/test_formatter.py.orig	2019-04-04 05:59:54.000000000 +0200
+++ reno-2.11.3/reno/tests/test_formatter.py	2020-01-13 20:36:59.774276596 +0100
@@ -12,7 +12,10 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 
 from reno import config
 from reno import formatter
--- reno-2.11.3/reno/tests/test_loader.py.orig	2019-04-04 05:59:54.000000000 +0200
+++ reno-2.11.3/reno/tests/test_loader.py	2020-01-13 20:37:14.794195227 +0100
@@ -16,7 +16,10 @@
 import textwrap
 
 import fixtures
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import six
 import yaml
 
--- reno-2.11.3/reno/tests/test_scanner.py.orig	2020-01-13 20:37:28.574120574 +0100
+++ reno-2.11.3/reno/tests/test_scanner.py	2020-01-13 20:37:30.737442188 +0100
@@ -25,7 +25,10 @@
 from dulwich import diff_tree
 from dulwich import objects
 import fixtures
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 from testtools.content import text_content
 
 from reno import config
--- reno-2.11.3/reno/tests/test_utils.py.orig	2019-04-04 05:59:54.000000000 +0200
+++ reno-2.11.3/reno/tests/test_utils.py	2020-01-13 20:37:40.040725121 +0100
@@ -12,7 +12,10 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-import mock
+try:
+    import mock
+except ImportError:
+    from unittest import mock
 import six
 
 from reno.tests import base
